prime-router/src/test/kotlin/fhirengine/utils/FhirTranscoderTests.kt

```diff
@@ -1,13 +1,41 @@
 package gov.cdc.prime.router.fhirengine.utils
 
 import assertk.assertThat
+import assertk.assertions.contains
+import assertk.assertions.isNotEmpty
 import assertk.assertions.isNotNull
+import org.hl7.fhir.r4.model.Bundle
+import org.junit.jupiter.api.BeforeEach
+import java.util.Date
 import kotlin.test.Test
 
 class FhirTranscoderTests {
+    private val testBundle = Bundle()
+    @BeforeEach
+    fun setup() {
```

Comment:
Yeah the reason I made it before each was to prevent someone from making a change to the bundle in any subsequent tests that would affect the ones I wrote. I'll just adde the redundant bundle set up code in the tests that need it